actor/echo/README.md

```diff
@@ -0,0 +1,5 @@
+A simple http echo server
```

Comment:
By github convention, READMEs generally have a heading, so please just put a `# ` markdown header indicator in front of this line.